src/mobx/stores/UserStore.ts

```diff
@@ -48,6 +48,19 @@ export default class UserStore {
     return rewardsValue + walletValue;
   }
 
+  get myAPR(): number {
+    const { vaults } = this.store;
+    const deposites: number[] = [];
+    vaults.vaultOrder.flatMap((vault) => {
+      const depositBalance = this.getBalance(vault.vaultToken);
+      deposites.push(depositBalance.price * depositBalance.balance);
+    });
+    const totalAPR =
```

Comment:
not sure what this maths is. 
for each vault you need to grab the apr the user of the vault is experiencing
weight that by the amount of deposit assets into the pool and add those values in an averaged fasion.

Reply:
Vault APR * ( "user's balance in vault" / "total user balances in all vaults").

1. calculate "total user balances in all vaults" 
2. loop through all vaults the user has a balance in and SUM the above formula up to get the APR for the user